### PR TITLE
Feature Contact

### PR DESCRIPTION
**Pull request by LuisDavidRodriguez**
# New features Added
- Footer with the user form
- Contact icons added to the end of the form
- The users can now send information to the [Formspree site](https://formspree.io/forms/mjvlqgdz/submissions)

## New abilities learnt
- Peer to Peer programming
- Research about stylelint no-descending-specificity

### Research and issues
- There was an specific issue trying to use the same class in the first contact list, related to the [stylein no-descending-specificity](https://stylelint.io/user-guide/rules/list/no-descending-specificity/)  which according with the source can be ignored because stylein has some [DOM limitation](https://stylelint.io/user-guide/rules/list/no-descending-specificity/#dom-limitations) of course only if you are totally sure that is a false error.
> **Note:** To sort this error I just declared a new id again and a new style to the #contact-2  **I'm not very happy with that solution because i'm repeating css code**, I just want to **confirm with my codeReviewer** that indeed I can [ignore that bloks](https://stylelint.io/user-guide/ignore-code) using this rule /* stylelint-disable, no-descending-specificity */ and if so applying it to the next project.
- Coding partner formatted my Html code with **pretty Formatter**, my partner has installed a plugin **pretty Formatter** so when he cloned my branch he changed all my html code of my portfolio, personally I found the format that pretty Formatter added pretty difficult to read and visualizate, it added a new lines, hence I returned it to the previous format you can see the format difference in the commit of my partner. It is that right? it is mean my html code is not correctly formatted?
> That is why the index file, show to much lines changed from the commit of my partner to the final mine

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,231 +1,249 @@
...
+            <li class="li">GitLab</li>
+            <li class="li">Terminal</li>
+          </ul>
+        </article>
+      </section>
     </main>
-    
+
+    <footer class="contact-container">
+      <h3>I'm always interested in hearing about new projects, so if you'd like touch at please get in touch.</h3>
```
Comment (gtekle) [inline index.html:224]: - [ Optional ] The content of this HTML element is a paragraph. Thus, I believe using `<p>` tag is semantically more appropriate. If you would like to add a section heading you can add something like `Contact Me` heading. what do you think?

**Review by gtekle — APPROVED** [review]
Hi **@TEAM** 👋 , 

# STATUS: APPROVED 🎉 🎉 🎉 👏 👏 👏 💯 💪 

## Well done! You nailed it on first attempt! 🎆 🎆 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Clean and organized code
✔️ Perfect form input validation
✔️ Nice UI
✔️ Meaningful commit messages
✔️ Descriptive PR title and summary

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (LuisDavidRodriguez) [reply index.html:224]: Ok great thank you i will change it!

**Comment by LuisDavidRodriguez** [comment]
@gtekle Do you had the opportunity to check the comments that i wrote in the pull description about the pretty formatter?

what advice could you give me about that? 

and also about ignoring the stylint error?

**Pull request merged**